docs/source/examples/examples.md

```diff
@@ -22,5 +22,6 @@ PEC on a Braket simulator: Mirror circuits <pec_tutorial.md>
 PEC with Cirq: Learning representations <learning-depolarizing-noise.md>
 DDD with Cirq: Mirror circuits <ddd_tutorial.md>
 DDD with Qiskit: GHZ circuits <ddd_on_ibmq_ghz.md>
+GGI Summer School ZNE Hands-On Tutorial <ggi_summer_school_unsolved.md>
 The Mitiq paper code <mitiq-paper/mitiq-paper-codeblocks.md>
```

Comment:
```suggestion
GGI Summer School ZNE Hands-On Tutorial <ggi_summer_school_unsolved.md>
GGI Summer School ZNE Hands-On Tutorial (Solved) <ggi_summer_school_solved.md>
The Mitiq paper code <mitiq-paper/mitiq-paper-codeblocks.md>
```
I would suggest this change as a temporary solution. In this way the docs should build without errors and it will be easier to review the content of both notebooks. 
Once the content of both notebooks will be reviewed, we'll find a way to hide the solution notebook.

Reply:
I removed the solved md file after using `orphan`. This notebook is linked at the end of the unsolved notebook.

Reply:
good!